hasher-matcher-actioner/hmalib/common/configs/tests/test_tx_collab_config.py

```diff
@@ -0,0 +1,102 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+import unittest
+from threatexchange.exchanges.clients import ncmec
+from threatexchange.exchanges.clients.ncmec.hash_api import NCMECEnvironment
+
+from threatexchange.exchanges.impl.fb_threatexchange_api import (
+    FBThreatExchangeSignalExchangeAPI,
+)
+from threatexchange.exchanges.impl.ncmec_api import (
+    NCMECCollabConfig,
+    NCMECSignalExchangeAPI,
+)
+
+
+from hmalib.common.config import HMAConfig, create_config
+from hmalib.common.models.tests.ddb_test_common import HMAConfigTestBase
+
+from hmalib.common.configs.tx_apis import (
+    add_signal_exchange_api,
+    ToggleableSignalExchangeAPIConfig,
+)
+from hmalib.common.configs.tx_collab_config import create_config_for_api, get_all
+
+
+def get_all_apis():
+    return [
+        api.to_concrete_class() for api in ToggleableSignalExchangeAPIConfig.get_all()
+    ]
+
+
+class TXCollabConfigsTestCase(HMAConfigTestBase, unittest.TestCase):
```

Comment:
nit: Something I've learned after using py.test more is that unittest.TestCase is not needed for anything, and actually makes it easier to accidentally prevent access to py.test features. You can use fixtures any time you would use class state.

In general, I'd recommend not using unittest classes, and refactored away the class uses in the threatexchange library

Reply:
Agree that unittest.TestCase is not needed, but some of the work I have done with HMAConfigTestBase makes it easier to handle creating mock dynamodb instances and handle some things auto-magically. I'd prefer to continue using them here.

Short of re-doing all of that using fixtures, there is no middle ground that I can think of. Unless it is categorically unsafe to use TestCase, I'd want to continue using it. 

Also, since you marked as a nit, I'm aware of the severity of your stance. :P